news.html

```diff
@@ -6,18 +6,31 @@
 
 <div>
   {% for post in site.posts %}
-  <article class="pb1 measure-wide lh-copy">
-    <h2 class="mb2"><a href="{{ post.url }}">{{ post.title }}</a></h2>
-    <h4 class="mt0">{{ post.date | date: "%B %-d, %Y" }}</h4>
-    <div class="mb2">
-      <p class="mb0">{{ post.excerpt | strip_html }}</p>
-      <a href="{{ post.url }}" class="gray link hover-cmu-red"
-        >continue reading...</a
-      >
+  <div class="flex flex-row-ns flex-wrap">
+    <div class="{% if post.side_image %} w-70-ns w-100-s {% else %} w-100 {% endif %}">
+      <article class="lh-copy">
+        <h2 class="mb2"><a href="{{ post.url }}">{{ post.title }}</a></h2>
+        <h4 class="mt0">{{ post.date | date: "%B %-d, %Y" }}</h4>
+        <div class="mb2">
+          <p class="mb0">{{ post.excerpt | strip_html }}</p>
+          {% if post.content.size > post.excerpt.size %}
+          <a href="{{ post.url }}" class="gray link hover-cmu-red"
+            >continue reading...</a
+          >
+          {% endif %}
+        </div>
+        {% if post.studyLink %}
+        <a href="{{ post.studyLink }}" class="cta"> Participate </a>
+        {% endif %}
+      </article>
     </div>
-    {% if post.studyLink %}
-    <a href="{{ post.studyLink }}" class="cta"> Participate </a>
+    {% if post.side_image %}
+      <div class="w-30-ns w-100-s pl4">
```

Comment:
Let's move the image so that it is below the heading for the article and maybe a bit smaller so it is visually clear which article an image belongs to.

Reply:
It feels a bit small to me now.. ideally I'd want the image to be sized based on the text div's height, but it was tricky to get flexbox to do that. If you know how to do that we can go with that

<img width="973" alt="Screen Shot 2021-03-05 at 11 24 31 AM" src="https://user-images.githubusercontent.com/13065028/110143571-5d1a1b80-7da5-11eb-853e-60c3e3b53ab7.png">